mozilla/blurts-server
# MNTOR-3353: add refresh token logic to session management

# References:

Jira: MNTOR-3353



# Description
- Adding refresh token fetching logic to next-auth
- The expiry / tokens are stored in the db

# Test

## Test for refresh token

- run `db:migration` and make sure the `fxa_session_expiry` col exists
- If the column is empty, login and see that the session expiry should be an hour in the future
- Change the time to be some time in the past OR set it to null 
- Go to dashboard, watch the session refresh, come back to the table and check on the session field again, it should have refreshed
- The access token should have also been refreshed

## Test for refresh fail

- empty out the columns `fxa_session_expiry`, `fxa_access_token`, and `fxa_refresh_token` columns in the `subscribers` table
- navigate to either dashboard or settings
- it should automatically log the user out and prompt a relogin

# Checklist (Definition of Done)
